_rules/SC1-3-5-autocomplete-valid.md

```diff
@@ -64,114 +64,177 @@ The intent of this rule is to ensure that the `autocomplete` attribute can be us
 
 ### Passed
 
+#### Pass example 1
+
+Single autocomplete term.
+
 ```html
-<!-- single autocomplete term -->
 <input autocomplete="username" />
 ```
 
+#### Pass example 2
+
+Single autocomplete term for select.
+
 ```html
-<!-- single autocomplete term for select -->
 <select autocomplete="bday-month">
   <option>January</option>
   <option>...</option>
 </select>
 ```
 
+#### Pass example 3
+
+Autocomplete term, only valid for textarea.
+
 ```html
-<!-- Autocomplete term, only valid for textarea -->
 <textarea autocomplete="Street-Address"></textarea>
 ```
 
+#### Pass example 4
+
+Two autocomplete terms.
+
 ```html
-<!-- Two autocomplete terms -->
 <input autocomplete="Work EMail" />
 ```
 
+#### Pass example 5
+
+Autocomplete using section-*
+
 ```html
-<!-- Autocomplete using section-* -->
 <input autocomplete="section-partner email" />
 ```
 
+#### Pass example 6
+
+Triple autocomplete terms.
+
 ```html
-<!-- Tripple autocomplete terms -->
 <input type="text" autocomplete="section-primary billing street-address />
 ```
 
+#### Pass example 7
```

Comment:
There's something odd going on with the code highlighter here. Is that on us or is it Github?

Reply:
Seems to be a GitHub issue